backend/app/api/ws/local_executor_namespace.py

```
@@ -0,0 +1,703 @@
+            event: Event name
+            data: Event data
+        """
+        from app.core.socketio import get_sio
+
+        sio = get_sio()
+        task_room = f"task:{task_id}"
+
+        try:
+            await sio.emit(event, data, room=task_room, namespace="/chat")
+            logger.debug(f"[LocalExecutor] Forwarded {event} to room {task_room}")
+        except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)